src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCost = 40;
+  let total = days * dailyRentalCost;
+  const threeDayDiscount = 20;
+  const weeklyDiscount = 50;
+
+  if (days >= 7) {
+    total = total - weeklyDiscount;
```

Comment:
You can simplify this code by directly subtracting the `weeklyDiscount` from `total` without reassigning the value to `total`. For example, `total -= weeklyDiscount;` ✨